src/core/env/prod/terraform.tfvars

```diff
@@ -303,6 +303,9 @@ eventhubs = [{
   partitions        = 30
   message_retention = 7
   consumers         = []
+  iam_roles = {
+    "/subscriptions/ec285037-c673-4f58-b594-d7c480da4e8b/resourceGroups/io-p-sign-backend-rg/providers/Microsoft.Web/sites/io-p-sign-backoffice-func" = "Azure Event Hubs Data Receiver"
```

Comment:
Have you tried if the CI/CD can read this principal? I'm not sure if it is allowed because because is a cross subscription reference. If you can't use this reference we can add the principal_id

Reply:
`plan` [went well](https://dev.azure.com/pagopaspa/selfcare-iac-projects/_build/results?buildId=125660&view=logs&j=c21e8a0c-ac94-510d-27bc-1342ac1fef91&t=14eec3f0-4d0a-51da-fc41-2b934d664131&l=1048), but I cannot give a try to the `apply` because of the huge diffs showed by the plan - both in DEV and UAT

Reply:
A seguito di [questo test](https://github.com/pagopa/selfcare-infra/actions/runs/6744010032/job/18333094271), sembrerebbe che specificando il principal id dell'identity, l'assegnazione funzioni anche cross subscription